### ISSUE TITLE
sanitazing requested observables

### ISSUE THREAD

**Issue by wkotlarski** [FlexibleSUSY/FlexibleSUSY#71]
Should we check if the observable block in `FlexibleSUSY.m.in` makes sense? For now this is done inside of `GetRequestedObservables` for the case of effective couplings. The problem is that the function `GetRequestedObservables` is called multiple times. Printing a warning from it every time someone calls it for an unrelated observable seems a bad idea. If you agree, than were should we check the content if this block?

**Comment by Expander** [comment]
I agree, moving the sanity check out of the `GetRequestedObservables` function would be better. The new `CheckObservables` function could then be called early in the `MakeFlexibleSUSY[]` function (where the other sanity checks are done).
